service/src/main/java/uk/gov/hmcts/reform/fpl/model/order/HearingOrder.java

```diff
@@ -61,12 +60,16 @@ public static HearingOrder from(DocumentReference order, HearingBooking hearing,
 
     @JsonIgnore
     public boolean isRemovable() {
-        return APPROVED.equals(status);
+        return !SEND_TO_JUDGE.equals(status);
     }
 
     public String asLabel() {
-        return format("%s - %s", defaultIfNull(title, "Case management order"),
-            formatLocalDateToString(dateIssued, "d MMMM yyyy"));
-    }
+        if (APPROVED.equals(status)) {
+            return format("Sealed case management order issued on %s",
+                formatLocalDateToString(dateIssued, "d MMMM yyyy"));
+        }
 
+        return format("Draft case management order sent on %s",
+            formatLocalDateToString(dateSent, "d MMMM yyyy"));
```

Comment:
Is dateSent defintely safe to use, can it be null?

Reply:
`dateSent` is set to the current date while creating a draft order. so it cannot be null